weibo_spider/parser/util.py

```diff
@@ -107,6 +107,7 @@ def to_video_download_url(cookie, video_page_url):
 def string_to_int(string):
     """字符串转换为整数"""
     if len(string) == 0:
+        print("Warning: the input string is empty!")
```

Comment:
感谢贡献代码，个人认为这里print改成logger.warning更好一点，因为方便写日志。如果有不同看法，欢迎讨论。

Reply:
emm..I think the function named string_to_int is not good.
Because of there is no flag to notified caller that inputted string is right.
Maybe need like this(ref to libc):

```python
import string 
string.atoi(covert_string)
```
I don't know why not use the function we had, but according to Qt, there have a *ok to notify that covert progress have not error occurred.
```c++
int QString::toInt(QString numberOfString, bool *ok)
```
and, the logger.warning is better than current solution.

Reply:
string_to_int是将字符串转化数字的，微博评论点赞等的数量有时会包含'万'等，需要转化成数字。您的意思是要在这个方法内添加个是否成功的返回值，还是添加个参数告诉本方法?我不了解qt，搜索了一下，发现是可视化工具，qt的规范是要这样吗？期待您的解答，感谢。